`cargo build --release` fails

Before I get to the main topic, thank you for creating this project! ❤️ 

The log of `cargo build --release`:

```
...
   Compiling github-webhook-type-generator v0.5.1 (/path/to/github-webhook-rs/type-generator)
error[E0599]: no method named `find_cycle` found for reference `&DirectedAcyclicGraph<Node>` in the current scope
  --> type-generator/src/dag.rs:60:29
   |
60 |             return Err(self.find_cycle());
   |                             ^^^^^^^^^^ method not found in `&DirectedAcyclicGraph<Node>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `github-webhook-type-generator` (lib) due to previous error
```

--- a/type-generator/src/dag.rs
+++ b/type-generator/src/dag.rs
@@ -55,7 +55,8 @@ impl<Node: Copy + Hash + Eq + std::fmt::Debug> DirectedAcyclicGraph<Node> {
                 }
             }
         }
-        if cfg!(debug_assertions) && in_degree.values().any(|&i| i != 0) {
+        #[cfg(debug_assertions)]
+        if in_degree.values().any(|&i| i != 0) {
             // the graph has cycle
             return Err(self.find_cycle());
         }

